libs/src/utils/network.js

```diff
@@ -27,7 +29,21 @@ async function timeRequests (requests) {
     timeRequest(request)
   ))
 
-  return timings.sort((a, b) => a.millis - b.millis)
+  timings.sort((a, b) => {
```

Comment:
does it make sense to add this logic here? or maybe as a separate util to sort by version?

Reply:
yeah, thinking about its some more, I think it's a better idea to make this a separate function in the case we want to use the original method to time other requests.